bittensor/_subtensor/subtensor_impl.py

```diff
@@ -249,16 +193,11 @@ async def _submit_and_check_extrinsic(
         if not wait_for_inclusion and not wait_for_finalization:
             return True
         else:
-            if 'error' in response:
-                logger.error('Error in extrinsic: {}', response['error'])
-            elif 'finalized' in response and response['finalized'] == True:
-                return True
-            elif 'inBlock' in response and response['inBlock'] == True:
+            if response.is_success:
                 return True
-            else:
-                return False
+            return False
 
-    def is_subscribed(self, wallet: 'bittensor.wallet', ip: str, port: int, modality: int ) -> bool:
+    def is_subscribed(self, wallet: 'bittensor.wallet', ip: str, port: int) -> bool:
```

Comment:
This will eventually use the retry logic correct?

Reply:
Correct, that's my next PR.